reframe/core/schedulers/slurm.py

```diff
@@ -193,6 +196,21 @@ def _get_reservation_nodes(self):
         node_descriptions = completed.stdout.splitlines()
         return (SlurmNode(descr) for descr in node_descriptions)
 
+    def _get_excluded_node_names(self):
+        if not self.sched_exclude_nodelist:
+            return set()
+
+        command = 'scontrol show -o -a %s' % self.sched_exclude_nodelist
```

Comment:
The command should be `scontrol show node -o -a [nodelist]`.

Reply:
I assumed that we are getting the nodes with `scontrol show -o -a Nodes=[nodelist]` which also works.